dblog_feed_example.xml

```diff
@@ -0,0 +1,167 @@
+		<wfw:commentRss>https://dlang.org/blog/2018/03/19/dconf-2018-programme-open-registration/feed/</wfw:commentRss>
+		<slash:comments>0</slash:comments>
+	<post-id xmlns="com-wordpress:feed-additions:1">1493</post-id>	</item>
+		<item>
+		<title>User Stories: Funkwerk</title>
+		<link>https://dlang.org/blog/2018/03/14/user-stories-funkwerk/</link>
+		<comments>https://dlang.org/blog/2018/03/14/user-stories-funkwerk/#comments</comments>
+		<pubDate>Wed, 14 Mar 2018 14:06:32 +0000</pubDate>
+		<dc:creator><![CDATA[Michael Parker]]></dc:creator>
+				<category><![CDATA[Code]]></category>
+		<category><![CDATA[Community]]></category>
+		<category><![CDATA[Companies]]></category>
+		<category><![CDATA[Guest Posts]]></category>
+		<category><![CDATA[User Stories]]></category>
+
+		<guid isPermaLink="false">http://dlang.org/blog/?p=1479</guid>
+		<description><![CDATA[In this post, we cap off the Funkwerk series with the launch of a new feature we creatively call “User Stories”. Now and again, we’ll publish a post in which D users talk of their experiences with D, not about specific projects, but about the language itself. They’ll tell of things like their favorite features, why they use it, how it has changed the way they write code, or anything they’d like to say that expresses how they feel about programming in D.]]></description>
+		<wfw:commentRss>https://dlang.org/blog/2018/03/14/user-stories-funkwerk/feed/</wfw:commentRss>
+		<slash:comments>3</slash:comments>
+	<post-id xmlns="com-wordpress:feed-additions:1">1479</post-id>	</item>
+		<item>
+		<title>The D Language Foundation at Open Collective</title>
+		<link>https://dlang.org/blog/2018/03/12/the-d-foundation-at-open-collective/</link>
+		<comments>https://dlang.org/blog/2018/03/12/the-d-foundation-at-open-collective/#respond</comments>
+		<pubDate>Mon, 12 Mar 2018 14:20:54 +0000</pubDate>
+		<dc:creator><![CDATA[Michael Parker]]></dc:creator>
+				<category><![CDATA[Community]]></category>
+		<category><![CDATA[D Foundation]]></category>
+
+		<guid isPermaLink="false">http://dlang.org/blog/?p=1441</guid>
+		<description><![CDATA[In its work guiding the development of D and promoting its adoption, the D Language Foundation is driven primarily by donations big and small. The money comes in from different sources, the most visible being those listed on the website’s donation page, and is put to use in different ways. Today, the D Language Foundation is opening a new chapter in the donation story.]]></description>
+		<wfw:commentRss>https://dlang.org/blog/2018/03/12/the-d-foundation-at-open-collective/feed/</wfw:commentRss>
+		<slash:comments>0</slash:comments>
+	<post-id xmlns="com-wordpress:feed-additions:1">1441</post-id>	</item>

```

Comment:
Cool, thanks! We only need at most two items, though.

Reply:
I thought if we already test this, then we should do it with an actual real world example.